Readme.md

```diff
@@ -34,6 +34,10 @@ Here's a quick public tour of Open Library to get your familiar with the service
 
 ## Installation
 
+Use Gitpod, a free online dev environment for GitHub.
+
+[![Open In Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/internetarchive/openlibrary/)
+
 The development environment can be set up using the [Docker Instructions](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md). You can also watch the [video tutorial](https://archive.org/embed/openlibrary-developer-docs/openlibrary-docker-set-up.mp4) for a more detailed explanation.
```

Comment:
```suggestion
The development environment can be set up using the [Docker Instructions](https://github.com/internetarchive/openlibrary/blob/master/docker/README.md). You can also watch the [video tutorial](https://archive.org/embed/openlibrary-developer-docs/openlibrary-docker-set-up.mp4) for a more detailed explanation.

***Alternatively***, if you do not want to set up Open Library on your local computer, try Gitpod! This will let you work on Open Library entirely in your browser without having to install anything on your personal computer.

[![Open In Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/internetarchive/openlibrary/)
```

And can we also place a copy the GitPod badge at the top of the file with the other badges? That way someone who knows what GitPod is will be able to hit it up straight away.

Reply:
Since we've just started with this, let's make it as an alternative to our mail flow just in case folks hit strange issues.